Solutions/Dynatrace/Playbooks/Enrich_DynatraceApplicationSecurityAttackIncident/azuredeploy.json

```diff
@@ -0,0 +1,641 @@
+                                    "incidentArmId": "@triggerBody()?['object']?['id']",
+                                    "tagsToAdd": {
+                                        "TagsToAdd": "@variables('tags')"
+                                    }
+                                },
+                                "host": {
+                                    "connection": {
+                                        "name": "@parameters('$connections')['microsoftsentinel']['connectionId']"
+                                    }
+                                },
+                                "method": "put",
+                                "path": "/Incidents"
+                            }
+                        }
+                    }
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "microsoftsentinel": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('MicrosoftSentinelConnectionName'))]",
+                                "connectionName": "[variables('MicrosoftSentinelConnectionName')]",
+                                "id": "[subscriptionResourceId('Microsoft.Web/locations/managedApis', resourceGroup().location,'azuresentinel')]",
+                                "connectionProperties": {
+                                    "authentication": {
+                                        "type": "ManagedServiceIdentity"
+                                    }
+                                }
+                            },
+                            "azuremonitorlogs": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('AzureMonitorLogsConnectionName'))]",
+                                "connectionName": "[variables('AzureMonitorLogsConnectionName')]",
+                                "id": "[subscriptionResourceId('Microsoft.Web/locations/managedApis', resourceGroup().location,'azuremonitorlogs')]"
```

Comment:
Please check if azuremonitorlogs can also be a "ManagedServiceIdentity"

Reply:
> Please check if azuremonitorlogs can also be a "ManagedServiceIdentity"

Hi @manishkumar1991 ,
as far as I am aware its not supported? do you have a working example? I get the following error when trying to switch to managed identity

> {
    "status": "Failed",
    "error": {
        "code": "InvalidApiConnectionAlternativeParameters",
        "message": "The API connection 'azuremonitorlogs-EnrichDynatraceApplicationSecurityAttackIncident' has invalid inputs. Parameter value type cannot be set to 'Alternative' because managed identity is not supported in the API definition's connection parameters."
    }
}